test/pipetool.uts

```diff
@@ -9,13 +9,13 @@
 s = PeriodicSource("hello", 1, name="src")
 d1 = Drain(name="d1")
 c = ConsoleSink(name="c")
-tf = TransformDrain(lambda x:"Got %r" % x)
+tf = TransformDrain(lambda x: "Got %s" % x)
```

Comment:
Maybe you could leave '%r', and replace `x` by `x.decode()`?

Reply:
Well, it's more readable to use `%s`. Also, as we are in a test it does not really matter